scripts/stubsabot.py

```diff
@@ -308,28 +308,10 @@ def normalize(name: str) -> str:
 BRANCH_PREFIX = "stubsabot"
 
 
-async def suggest_typeshed_update(update: Update, session: aiohttp.ClientSession, action_level: ActionLevel) -> None:
-    if action_level <= ActionLevel.nothing:
-        return
-    title = f"[stubsabot] Bump {update.distribution} to {update.new_version_spec}"
-    async with _repo_lock:
-        branch_name = f"{BRANCH_PREFIX}/{normalize(update.distribution)}"
-        subprocess.check_call(["git", "checkout", "-B", branch_name, "origin/master"])
-        with open(update.stub_path / "METADATA.toml", "rb") as f:
-            meta = tomlkit.load(f)
-        meta["version"] = update.new_version_spec
-        with open(update.stub_path / "METADATA.toml", "w") as f:
-            tomlkit.dump(meta, f)
-        subprocess.check_call(["git", "commit", "--all", "-m", title])
-        if action_level <= ActionLevel.local:
-            return
-        somewhat_safe_force_push(branch_name)
-        if action_level <= ActionLevel.fork:
-            return
-
-    body = "\n".join(f"{k}: {v}" for k, v in update.links.items())
-
-    stubtest_will_run = not meta.get("stubtest", {}).get("skip", False)
+def get_pr_body(update: Update, metadata: dict[str, Any]) -> str:
+    body = "Project links:\n - "
```

Comment:
nit: will have stray dash if there aren't update.links. the update.links are labelled and filtered to only changelog + release + home page, so not sure we need this